src/Data/NodeResolver.php

```diff
@@ -115,7 +129,7 @@ public function resolve_uri( string $uri, $extra_query_vars = '' ) {
 		$this->wp->query_vars = [];
 		$post_type_query_vars = [];
 
-		if ( isset( $parsed_url['query'] ) && '/' === $parsed_url['path'] ) {
+		if ( isset( $parsed_url['query'] ) && ( empty( $parsed_url['path'] ) || '/' === $parsed_url['path'] ) ) {
```

Comment:
if `$uri = '?myparam';` then `$parsed_url['path']` is not smart enough to know it refers to the home page.